autoconfigure/README.md

```diff
@@ -0,0 +1,36 @@
+# Configuration
+
+## Storage configurations
+
+| Configuration | Description | Default |
+|---------------|-------------|---------|
+| `KAFKA_STORE_SPAN_CONSUMER_ENABLED` | Enable partitioning of incoming span batches into individual spans keyed by trace ID. | `true` |
+| `KAFKA_STORE_AGGREGATION_ENABLED` | Enable aggregation spans into traces and then into dependency links. Currently this has to be initiated with span consumer or storage, or initiate manually via `curl /health` | `true` |
+| `KAFKA_STORE_SPAN_STORE_ENABLED` | Enable storage for spans, traces, dependencies, tags, and service names. | `true` |
+| `KAFKA_STORE_BOOTSTRAP_SERVERS` | Kafka bootstrap servers | `localhost:9092` |
+| `KAFKA_STORE_ENSURE_TOPICS` | Ensure topics are created if don't exist. Is recommended to create them manually, to define partition and replication according to your environment. | `true` |
+| `KAFKA_STORE_COMPRESSION_TYPE` | Compression type used to store data in Kafka topics | `NONE` |
+| `KAFKA_STORE_DIRECTORY` | Root path where Zipkin stores tracing data | `/tmp/zipkin` |
```

Comment:
look at other environment variables in products that configure where they keep data. see if there's some naming trends and if this name could be tightened up or not.

Reply:
will go for `KAFKA_STORAGE_DIR` similar to kafka logs.dir, or kafka streams store.dir config.